src/bin/pg_autoctl/pgsql.c

```diff
@@ -1304,12 +1313,11 @@ pgsql_get_wal_lag_from_standby(PGSQL *pgsql, int64_t *walLag)
 		return false;
 	}
 
-	*walLag = context.bigint;
+	strlcpy(receivedLSN, context.strVal, bufferLength-1);
```

Comment:
This usage of `bufferLength-1` is worrying me a lot, can you double-check?

Reply:
removed - 1